src/main/java/org/folio/oaipmh/helpers/AbstractGetRecordsHelper.java

```diff
@@ -108,37 +110,44 @@
                                                            org.folio.rest.tools.client.Response instancesResponse) {
     requiresSuccessStorageResponse(instancesResponse);
 
-    JsonArray instances = storageHelper.getItems(instancesResponse.getBody());
-    Integer totalRecords = storageHelper.getTotalRecords(instancesResponse.getBody());
+    JsonObject body = instancesResponse.getBody();
+    JsonArray instances = storageHelper.getItems(body);
+    Integer totalRecords = storageHelper.getTotalRecords(body);
+
+    if (logger.isDebugEnabled()) {
+      logger.debug(String.format("%d entries retrieved out of %d", instances.size(), totalRecords));
```

Comment:
It's better to use version of debug method that takes parameters instead of building log string in advance. In this case, no check for enabled debug is required.

Reply:
Done